app/views/admin/project_steps/_form.html.slim

```diff
@@ -13,32 +18,32 @@ div.panel-body class=('new-record' if step.new_record?)
 
     = f.error_notification
     = f.translatable_inputs do |l|
-      = f.input "summary_#{l}".to_sym, wrapper: :step_form_left_12,
+      = f.input "summary_#{l}".to_sym,
         input_html: {data: {translatable: 'common.summary'}},
-        placeholder: I18n.t('common.summary', locale: l)
-      = f.input "details_#{l}".to_sym, as: :text, wrapper: :step_form_left_12,
+        placeholder: I18n.t('common.summary', locale: l),
+        label: I18n.t('common.summary', locale: l)
+      = f.input "details_#{l}".to_sym, as: :text,
         input_html: {rows: 3, data: {translatable: 'common.details'}},
-        placeholder: I18n.t('common.details', locale: l)
-
-    .step_form_right.col-sm-4
-      = f.input :step_type_value, label: 'Type',
-        collection: ProjectStep.step_type_option_set.translated_list,
-        wrapper: :step_form_right_9,
-        include_blank: '---',
-        wrapper_html: {class: 'clearfix'},
-        input_html: {class: 'type'}
-      = f.input :scheduled_start_date, label: 'Scheduled', as: :date_picker, wrapper_html: {class: 'clearfix'},
-        wrapper: :step_form_right_9
-      = f.input :actual_end_date, label: 'Completed', as: :date_picker, wrapper_html: {class: 'clearfix'},
-        wrapper: :step_form_right_9
-      .form-group.date.optional.project_step_actual_end_date
-        = label_tag :is_finalized, "Finalized", class: "date optional col-sm-3 checkbox-label"
-        .col-sm-9
-          - if step.is_finalized_locked?
-            = step.is_finalized ? t("reply_yes") : t("reply_no")
-            =< "(#{t('locked')})"
-          - else
-            = f.check_box :is_finalized, class: "step_form_right_9 finalized"
+        placeholder: I18n.t('common.details', locale: l),
+        label: I18n.t('common.details', locale: l)
+
+    = f.input :step_type_value,
+      collection: ProjectStep.step_type_option_set.translated_list,
+      include_blank: '---',
+      input_html: {class: 'type'}
+    = f.input :scheduled_start_date, as: :date_picker
+    = f.input "duration" do
+      = f.input_field :scheduled_duration_days, input_html: {class: 'duration'}
```

Comment:
Can you get rid of these explicit classes for duration and type? And perhaps finalized? Or is there some reason they have to stay?